bedrock/firefox/views.py

```diff
@@ -242,6 +242,106 @@ def send_to_device_ajax(request):
+    else:
+        view = all_downloads(request, platform, channel)
+
+    return view
+
+
+def all_downloads_unified(request):
+    product_android = firefox_android
+    product_desktop = firefox_desktop
+
+    # Human-readable product labels
+    products = OrderedDict([
+        ('desktop_release', 'Firefox'),
+        ('desktop_beta', 'Firefox Beta'),
+        ('desktop_developer', 'Firefox Developer Edition'),
+        ('desktop_nightly', 'Firefox Nightly'),
+        ('desktop_esr', 'Firefox Extended Support Release'),
+        ('android_release', 'Firefox Android'),
+        ('android_beta', 'Firefox Android Beta'),
+        ('android_nightly', 'Firefox Android Nightly'),
+    ])
+
+    channel_release = 'release'
+    channel_beta = 'beta'
+    channel_alpha = 'alpha'
+    channel_nightly = 'nightly'
+    channel_esr = 'esr'
+    channel_esr_next = 'esr_next'
+
+    latest_release_version_desktop = product_desktop.latest_version(channel_release)
+    latest_beta_version_desktop = product_desktop.latest_version(channel_beta)
+    latest_developer_version_desktop = product_desktop.latest_version(channel_alpha)
+    latest_nightly_version_desktop = product_desktop.latest_version(channel_nightly)
+    latest_esr_version_desktop = product_desktop.latest_version(channel_esr)
+    latest_esr_next_version_desktop = product_desktop.latest_version(channel_esr_next)
+
+    latest_release_version_android = product_android.latest_version(channel_release)
+    latest_beta_version_android = product_android.latest_version(channel_beta)
+    latest_nightly_version_android = product_android.latest_version(channel_nightly)
+
+    context = {
+        'products': products.items(),
+
+        'desktop_release_platforms': product_desktop.platforms(channel_release, True),
+        'desktop_release_full_builds': product_desktop.get_filtered_full_builds(channel_release, latest_release_version_desktop),
+        'desktop_release_channel_label': product_desktop.channel_labels.get(channel_release, 'Firefox'),
+        'desktop_release_latest_version': latest_release_version_desktop,
+
+        'desktop_beta_platforms': product_desktop.platforms(channel_beta, True),
+        'desktop_beta_full_builds': product_desktop.get_filtered_full_builds(channel_beta, latest_beta_version_desktop),
+        'desktop_beta_channel_label': product_desktop.channel_labels.get(channel_beta, 'Firefox'),
+        'desktop_beta_latest_version': latest_beta_version_desktop,
+
+        'desktop_developer_platforms': product_desktop.platforms(channel_alpha, True),
+        'desktop_developer_full_builds': product_desktop.get_filtered_full_builds(channel_alpha, latest_developer_version_desktop),
+        'desktop_developer_channel_label': product_desktop.channel_labels.get(channel_alpha, 'Firefox'),
+        'desktop_developer_latest_version': latest_developer_version_desktop,
+
+        'desktop_nightly_platforms': product_desktop.platforms(channel_alpha, True),
```

Comment:
Should be `channel_nightly` instead of `channel_alpha` here right?

Reply:
Good catch!